tools/suggest-grid.mjs

```diff
@@ -636,6 +636,20 @@ async function main({ preserve, except, apply, seed }) {
     </ul>
     <p>Command-line command:</p>
     <pre><code>${cli.cmd}</code></pre>`);
+
+  // Generate JSON representing this allocation of sessions to rooms and slots to
+  // initialize the schedule
+  logIndent(2, '<h2>Raw Schedule</h2>');
+  logIndent(2, '<pre>');
+  console.log(sessions.map(s => (
+	{
+	    number: s.number,
+	    room: s.room,
+	    slot: s.slot
+	}
+  )));
```

Comment:
Ah, no, this isn't going to output JSON. It will output something that looks like JSON, but is just a default serialized version of the array intended for debug. Double quotes will be missing in particular. To output proper JSON, `JSON.stringify` needs to be used.

```suggestion
  console.log(JSON.stringify(sessions.map(s => (
	{
	    number: s.number,
	    room: s.room,
	    slot: s.slot
	}
  )), null, 2);
```